website/addons/base/views.py

```diff
@@ -480,6 +512,7 @@ def addon_deleted_file(auth, node, **kwargs):
 
     ret = serialize_node(node, auth, primary=True)
     ret.update(rubeus.collect_addon_assets(node))
+    retError = ERROR_MESSAGES[kwargs.get('error_type')].format(file_name=file_name, provider=trashed.provider)
```

Comment:
Running this locally and trying to access a deleted file throws a ```KeyError``` for me - looks like ```error_type``` isn't showing up in ```kwargs``` so is defaulting to ```None```